CONTRIBUTING.md

```diff
@@ -242,13 +242,16 @@ and concatenate the commit messages of the commits before merging.
 Changelog
 ---------
 
-When opening a pull request, you should write a changelog entry
-(or more in case of multiple independent changes).
-This is done by adding files in the `changelog.d` directory.
+When opening a pull request with a user-visible change, you should write a changelog entry
+(or more in case of multiple independent changes) -- the desciption will end up in
```

Comment:
— ← em dash instead of `--` maybe (like you did below)?

Reply:
thanks, already fixed!